docs/DeveloperGuide.md

```diff
@@ -409,7 +446,49 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
       Use case ends.
 
-      
+**UC08 - User adds a new policy**    
+
+**MSS**
+
+1. User chooses to add a new policy with a name and description.
+2. System gives success message.
+    
+    Use case ends.
+
+**Extensions**
+
+* 1a. User enters wrong add policy command format.
+
+    * 1a1. System gives corresponding error message.
+    
+        Use case ends.
+        
+* 1b. User attempts to add an existing policy.
+
+    * 1b1. System gives corresponding error message.
+    
+        Use case ends.
+
+**UC09 - User clears policy list**    
```

Comment:
Perhaps we can say User clears the list of existing policies. Or we can add what is the policy list in the glossary too